### PR TITLE
977 fix codacy warnings

### PR DESCRIPTION
**Pull request by tsvetaevAlex**
Task #977 :
https://app.codacy.com/manual/jdi-testing/jdi-light/issues?&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6W251bGxdfSx7ImlkIjoiQ2F0ZWdvcnkiLCJ2YWx1ZXMiOltudWxsXX0seyJpZCI6IkxldmVsIiwidmFsdWVzIjpbIldhcm5pbmciXX0seyJpZCI6IlBhdHRlcm4iLCJ2YWx1ZXMiOls1MjYyXX0seyJpZCI6IkF1dGhvciIsInZhbHVlcyI6W251bGxdfV0=

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4524798&bid=15237172)

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/TableSteps.java [hunk 1]
```diff
@@ -24,17 +23,28 @@
...
+    {
+        return lastTable = getUI(name, Table.class);
+    }
+
+    //#region "When"
     @When("^(?:I |)click the cell \\(([^\\)]*)\\) in the \"([^\"]*)\"(?: table|)$")
     public void selectCell(String cellPair, String name) {
         cell(name, cellPair).click();
     }
-    //endregion
```
Comment (RomE-HK) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/TableSteps.java]: This statement was lost

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/TableSteps.java [hunk 2]
```diff
@@ -24,17 +23,28 @@
...
 
-    //#region Then
+    //#region "When"
+    /*
+    @When("^(?:I |)click the cell \\(([^\\)]*)\\) in the \"([^\"]*)\"(?: table|)$")
+    public void selectCell(String cellPair, String name) {
+        cell(name, cellPair).click();
+    }*/
+    //endregion "When"
```
Comment (RomE-HK) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/TableSteps.java]: Is this commented duplicate still needed?

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4524798&bid=15237172)

**Pull request merged**